.github/workflows/staging.yml

```diff
@@ -30,23 +30,17 @@ jobs:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
     
-    # Check-out the tooling
-    - uses: actions/checkout@v2
-      with:
-        repository: silnrsi/sldrtools
-        path: tools
-    
     # Install SLDR tools
     - name: Setup sldrtools
       working-directory: tools
-      run: pip install .
+      run: pip install 'git+https://github.com/silnrsi/sldrtools'
```

Comment:
Why hard code the URL here? This will prohibit the workflow from working for branches and forks.

Reply:
Nevermind I see you're using tooling from one repo to manipulate a different one, sorry I missed that.

Reply:
Sorry I have no idea how you got added as a review this is an internal development PR to test everything works before merging.